package.json

```diff
@@ -72,9 +72,9 @@
     "eslint-plugin-vitest": "^0.3.20",
     "prettier-plugin-jsdoc": "^1.3.0",
     "release-plan": "^0.11.0",
-    "typescript": "^5.3.3",
-    "vite": "^6.0.3",
-    "vitest": "^1.1.0"
+    "typescript": "^5.8.3",
+    "vite": "^6.3.5",
+    "vitest": "^3.2.2"
```

Comment:
Not too familiar with `vitest`'s release history. I assumed that we're good as long as existing tests can run and pass.